nbs/15_gsa_gls/30-real_phenotypes/10-summarize-phenomexcan.ipynb

```diff
@@ -0,0 +1,2001 @@
+{
```

Comment:
<p>what is GLS?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/phenoplier/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Generalized Least Squares. I updated the description.